### PR TITLE
An example: howto mock timestamp

### PR DESCRIPTION
**Pull request by tisnik**
An example: howto mock timestamp

### CODE REVIEW

File: tests/unit/app/endpoints/test_feedback.py [hunk 1]
```diff
@@ -45,16 +46,19 @@ def test_get_feedback_status(feedback_location):
     assert not feedback.is_feedback_enabled()
 
 
-def test_store_feedback(feedback_location):
+@patch('ols.app.endpoints.feedback.datetime')
```
Comment (bparees) [inline tests/unit/app/endpoints/test_feedback.py:49]: can't we just do:

```suggestion
@patch('ols.app.endpoints.feedback.datetime.utcnow', return_val=datetime(2000,1,1,1,23,45))
```
and avoid the lambda below?

Comment (tisnik) [reply tests/unit/app/endpoints/test_feedback.py:49]: `utcnow` returns `callable`, not directly a value

Comment (bparees) [reply tests/unit/app/endpoints/test_feedback.py:49]: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

```
This is like [now()](https://docs.python.org/3/library/datetime.html#datetime.datetime.now), but returns the current UTC date and time, as a naive [datetime](https://docs.python.org/3/library/datetime.html#datetime.datetime) object.
```

??

**Comment by tisnik** [comment]
no longer needed -> closing it

**Pull request closed without merging**